spec/features/resource_manager_modifies_appointments_spec.rb

```diff
@@ -138,7 +137,7 @@ def and_there_is_a_bookable_slot_for_a_guider
     )
   end
 
-  def then_they_can_see_the_bookable_slot
+  def then_they_can_see_the_bookable_slot # rubocop:disable Metrics/AbcSize
```

Comment:
@benlovell Fair enough. I would do that as another PR though, because of all the deletions that will have to happen.